robot_tests/initrd.robot

```diff
@@ -32,7 +32,8 @@ Rootfs should be set up
     Directory Should Exist  /root
     Directory Should Exist  /usr
     Directory Should Exist  /sbin
```

Comment:
This is the main reason why I have to create another CR. Since you changed it for /lib already, I guess it will fail now?

```suggestion
    Directory Should Exist  /usr/sbin
```